plugins/modules/firewalld.py

```diff
@@ -213,6 +213,7 @@
 '''
 
 from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.parsing.convert_bool import boolean
```

Comment:
```suggestion
from ansible.module_utils.parsing.convert_bool import boolean
```
that's great that you found and use this function!